README.md

```diff
@@ -125,6 +125,9 @@ for information about how to contribute recipes, features, tests, and community-
 
 ## Alternatives
 
+- [rocker/binder](https://rocker-project.org/images/versioned/binder.html) -
+  From the R focused [rocker-project](https://rocker-project.org),
+  lets you run both RStudio and Jupyter either standalone or in a JupyterHub
```

Comment:
Fixed style to match the markdown style used in this project, otherwise LGTM.

I would appreciate it if you created a similar PR in the `rocker-project` repo 🙂

Reply:
Thanks @mathbunnyru! I've put it on my list and will make such a PR after the holidays :)

Reply:
@yuvipanda a friendly reminder 🙂

Reply:
Great, thanks!